### ISSUE TITLE
Feature: 1.19 Port

### ISSUE THREAD

**Issue by bconlon1** [Gilded-Games/The-Aether#660]
- [x] Skyroot Bucket of Tadpole (lore, texture, programmer art)
- [x] Skyroot Boat with Chest (lore, texture, programmer art)
- [x] `#convertable_to_mud`, add `aether_dirt`.
- [x] `#sculk_replaceable`, add `holystone`, `quicksoil`? Look for other blocks as well.
- [x] `#snaps_goat_horn`, add Aether logs (create new tag for Aether logs).
- [x] Investigate `#snow_layer_can_survive_on` and `#snow_layer_cannot_survive_on`
- [x] `#chest_boats`, add Skyroot Boat with Chest
- [x] `#frog_food` add Swets
- [x] `#frog_food` add Sentries (they will explode).
- [x] Aether dimension biome tag (`#is_aether`)
- [x] Waterloggable Aether Leaves
- [x] Investigate fence model in item frame (may be fine if its parented properly).
- [x] New `monster_spawn_block_light_limit` and `monster_spawn_light_level` values for DimensionTypes will need to be specified.
- [x] Forge patch for using a custom seed for the dimension should no longer be necessary.
- [x] Update resource pack format.
- [x] Review predicates for loot/advancements to make sure nothing has broken with them.
- [x] Replace Random with RandomSource.
- [x] Make use of changes to DeferredRegister.
- [x] Investigate new worldgen datagen.
- [x] Fix world preview system.
- [x] Fix worldgen/noise settings.
- [x] Update CircleCI scripts accordingly and reset ANTS.
- [x] Make 1.19 branch the main branch and lock it.

**Comment by bconlon1** [comment]
Blocked by https://github.com/MinecraftForge/MinecraftForge/issues/8676.

**Comment by bconlon1** [comment]
https://github.com/MinecraftForge/MinecraftForge/issues/8689 is probably also a concern.

**Comment by bconlon1** [comment]
https://github.com/MinecraftForge/MinecraftForge/pull/8687

**Comment by bconlon1** [comment]
https://github.com/MinecraftForge/MinecraftForge/issues/8762

**Comment by bconlon1** [comment]
Another PR required for us to move forward with finishing datagen fixes: https://github.com/MinecraftForge/MinecraftForge/pull/8761
